crates/services/txpool_v2/src/tests/pool.rs

```diff
@@ -0,0 +1,1627 @@
+#![allow(non_snake_case)]
+
+use crate::{
+    config::Config,
+    error::Error,
+    tests::context::{
+        check_tx,
+        check_tx_with_gas_price,
+        check_unwrap_tx,
+        check_unwrap_tx_with_gas_price,
+        create_coin_output,
+        create_contract_input,
+        create_contract_output,
+        create_message_predicate_from_message,
+        IntoEstimated,
+        PoolContext,
+        TEST_COIN_AMOUNT,
+    },
+};
+use fuel_core_types::{
+    blockchain::header::ConsensusParametersVersion,
+    fuel_asm::{
+        op,
+        RegId,
+        Word,
+    },
+    fuel_tx::{
+        input::coin::CoinPredicate,
+        Address,
+        AssetId,
+        BlobBody,
+        BlobId,
+        BlobIdExt,
+        ConsensusParameters,
+        Contract,
+        Finalizable,
+        Input,
+        Output,
+        PredicateParameters,
+        Transaction,
+        TransactionBuilder,
+        TxParameters,
+        UniqueIdentifier,
+        UtxoId,
+    },
+    fuel_types::ChainId,
+    fuel_vm::checked_transaction::{
+        CheckError,
+        Checked,
+        CheckedTransaction,
+        IntoChecked,
+    },
+    services::txpool::PoolTransaction,
+};
+use std::vec;
+
+const GAS_LIMIT: Word = 100000;
+
+// TODO: Move out of tests
```

Comment:
Unaddressed TODO, can you resolve it? It's not obvious to me where this should be moved. The function is only used within this file as far as I can see.

Reply:
I removed the todo for now I think it's ok that's it's there